plugins/generator-1.20.4/neoforge-1.20.4/templates/livingentity/livingentity.java.ftl

```diff
@@ -655,13 +641,13 @@ public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements R
 	</#if>
 
 	<#if data.breatheUnderwater?? && (hasProcedure(data.breatheUnderwater) || data.breatheUnderwater.getFixedValue())>
-	@Override public boolean canBreatheUnderwater() {
+	@Override public boolean canDrownInFluidType(FluidType type) {

```

Comment:
canBreatheUnderwater is still used by vanilla. Deprecation on canBreatheUnderwater is for callers, not for overriders

Reply:
It is made final (from what I saw in build logs on GitHub).

Reply:
Oh, I see:

![image](https://github.com/MCreator/MCreator/assets/16374228/b5ed71d6-75a1-41b2-aa64-73130ccee789)